libs/api/src/lib/memberContext.ts

```diff
@@ -694,14 +700,66 @@ export class MemberContext implements MemberContextInterface {
       throw new InternalError()
     }
 
-    const invoice = await this.renewSubscriptionForUser({subscription})
+    if (startsAt < now || paidUntil) {
```

Comment:
@tomaszdurka do you checked that only admins can pass different startAt and paidUntil?